src/meta/play-meta.js

```diff
@@ -125,6 +126,18 @@ export const plays = [
     cover: 'https://res.cloudinary.com/atapas/image/upload/v1650866465/demos/cover_y20bzk.png',
     blog: '',
     video: ''
+  }, {
+    id: 'pl-expanding-cards', 
+    name: 'Expanding-Cards', 
+    description: 'this project shows bunch of striped cards on screen if you want to show the card. you just have to click to expand it',
+    component: () => {return <ExpandingCards />},
+    path: '/plays/expanding-cards',
+    level: 'Beginner',
+    tags: '',
+    github: 'deepak8717',
```

Comment:
Seems your GitHub account doesn't have your name updated? The name with this id returns null. It will be better you set your name on your GitHub account, then it will appear automatically in your play header.